### DISCUSSION TITLE
How To Send An Image File?

### DISCUSSION THREAD

**Discussion by pierreoramassyntegra** [WhiskeySockets/Baileys#354] [General]
I want to know how To Send An Image File?

**Comment by Skidy89** [comment]
sock.sendMessage(jid, { image: fs.readFileSync('/image.jpg') })

anyways I recommend that before opening any unnecessary thread you read the documentation and enter the baileys discord group 

:>

**Reply by Riders004** [reply]
I agree with you

**Comment by Riders004** [comment]
> sock.sendMessage(jid, { image: fs.readFileSync('/image.jpg') })
> 
> anyways I recommend that before opening any unnecessary thread you read the documentation and enter the baileys discord group
> 
> :>

I agree with you
